### ISSUE TITLE
Discriminative feature generation for classification of imbalanced data. (arXiv:2010.12888v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#45]
Discriminative feature generation for classification of imbalanced data. (arXiv:2010.12888v1 [cs.CV])<br>
https://ift.tt/3dXXK89<br>
<p>The data imbalance problem is a frequent bottleneck in the classification performance of neural networks. In this paper, we propose a novel supervised discriminative feature generation (DFG) method for a minority class dataset. DFG is based on the modified structure of a generative adversarial network consisting of four independent networks: generator, discriminator, feature extractor, and classifier. To augment the selected discriminative features of the minority class data by adopting an attention mechanism, the generator for the class-imbalanced target task is trained, and the feature extractor and classifier are regularized using the pre-trained features from a large source data. The experimental results show that the DFG generator enhances the augmentation of the label-preserved and diverse features, and the classification results are significantly improved on the target task. The feature generation model can contribute greatly to the development of data augmentation methods through discriminative feature generation and supervised attention methods.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
